apps/ledger-live-mobile/src/screens/CustomImage/Step0Welcome.tsx

```diff
@@ -8,18 +8,12 @@ import CustomImageBottomModal from "../../components/CustomImage/CustomImageBott
 import BottomButtonsContainer from "../../components/CustomImage/BottomButtonsContainer";
 import { ScreenName } from "../../const";
 import { CustomImageNavigatorParamList } from "../../components/RootNavigator/types/CustomImageNavigator";
-import { loadImageSizeAsync } from "../../components/CustomImage/imageUtils";
 import imageSource from "./assets/welcome.png";
-import { ImageDimensions } from "../../components/CustomImage/types";
 
-let imageDimensions: ImageDimensions | null = null;
-loadImageSizeAsync(Image.resolveAssetSource(imageSource).uri)
-  .then(res => {
-    imageDimensions = res;
-  })
-  .catch(e => {
-    console.error(e);
-  });
+const imageDimensions = {
+  height: Image.resolveAssetSource(imageSource).height,
+  width: Image.resolveAssetSource(imageSource).width,
```

Comment:
actually don't know if `resolveAssetSource` is expensive but: should you save the result and only call `resolveAssetSource` once ?

Reply:
This is only called once in the JS runtime / reload lifecycle (I don't know how to call this)